### PR TITLE
feat: add external ids to the schema inside Entity

### PR DESCRIPTION
**Pull request by ilkyazAlts**
Updated schema - added external Ids to Entity. Thanks for reviewing

### CODE REVIEW

File: aylien/v5/news/api-docs.yaml [hunk 1]
```diff
@@ -4482,6 +4482,20 @@ components:
           $ref: "#/components/schemas/EntityInText"
         title:
           $ref: "#/components/schemas/EntityInText"
+        external_ids:
+          description: External Id(s) of an entity along with the domain information
+          items:
+            $ref: "#/components/schemas/ExternalIds"
+          type: array
+      type: object
+    ExternalIds:
```
Comment (FrontSide) [inline aylien/v5/news/api-docs.yaml:4491]: Since this component only holds one external entity, it should probably be called just "ExternalId"

**Review by FrontSide — APPROVED** [review]

**Comment by fcibecchini** [comment]
I see we had to update 6 (!) yaml files last time we had to add a new object to the interface (https://github.com/AYLIEN/apis/pull/73/files). I don't remember why is that but we might need to do the same now? @FrontSide @marcinaylien

**Comment by FrontSide** [comment]
> I see we had to update 6 (!) yaml files last time we had to add a new object to the interface (https://github.com/AYLIEN/apis/pull/73/files). I don't remember why is that but we might need to do the same now? @FrontSide @marcinaylien

I believe there is v1, v4 and v5. 
v4 is for the SDKs with the old entities, so I don't think we'll need to update that.
v5 is for the new entities SDKs, that's the one being updated here.
And v1 I think is actually what is used by api-news and contains compatibility for both old and new entities so we'll probably also need to update it there. @ilkyazAlts

**Review by FrontSide — APPROVED** [review]

**Comment by marcinaylien** [comment]
Eoin is looking to get rid of SDKs at all.
And we cannot get rid of this v1 and v4 because some of our users are still using the old version of entities...
We need to rise this during the retro.

**Review by marcinaylien — APPROVED** [review]

**Pull request merged**